### ISSUE TITLE
Spotify login

### ISSUE THREAD

**Issue by Varun23102002** [Sangwan5688/BlackHole#750]
Unable to login in Spotify top charts

**Comment by RupeshHacker** [comment]
> Unable to login in Spotify top charts

No its working in my device. can you tell which browser you are using

**Comment by shxxxam** [comment]
Yeah its working fine! @Varun23102002 what are you actually doing? can you tell us in which specific step you are facing the issue?
 _like for me when I just clicked on "Sign in to Spotify" and selected browser(Brave in my case) it just went smoothly without any problem._
